app/models/user.rb

```diff
@@ -103,6 +103,8 @@ class User < ApplicationRecord
   acts_as_follower
 
   has_one :profile, dependent: :destroy
+  has_one :notification_setting, class_name: "Users::NotificationSetting", dependent: :destroy
```

Comment:
Create two new tables with a relationship to the Users table.

Reply:
Will the settings have callbacks or anything else that requires us to instantiate objects or can we just delete them straight in the DB with `dependent: :delete`?

Reply:
I'm not sure as yet, I feel like this will come out in the next PR when I get into the details of the next PR (i.e. porting data etc.), unless you think that's a detail that needs to be determined upfront and will affect this PR.

Reply:
Nope, not important, just maybe keep at a note for yourself and switch it to `:delete` if we don't need actual objects to be instantiated (which I believe we won't).